### ISSUE TITLE
General Question - 3D Computer Vision Tasks in zju3dv lab

### ISSUE THREAD

**Issue by Dipankar1997161** [zju3dv/EasyMocap#335]
Hello @chingswy

This is a different question which is not part of Easymocap, however, I believe you could respond.

I have been assigned another project,  where I need to do the same **3D Reconstruction but for Animals, particularly Deer.**
So, I was wondering, if "zju3dv Lab" supports any such operations regarding animal reconstruction. It will be helpful.

In case you are aware of any other source or lab, kindly inform me. I look forward to hearing.

Thank you once again

**Comment by chingswy** [comment]
I believe this issue is relatively straightforward. Once you implement keypoint detection for animals, parameterize the model fitting, and ensure compatibility with the neural body, you can refer to the relevant paper "Artemis: Articulated Neural Pets with Appearance and Motion Synthesis" for further guidance.

**Comment by Dipankar1997161** [comment]
> I believe this issue is relatively straightforward. Once you implement keypoint detection for animals, parameterize the model fitting, and ensure compatibility with the neural body, you can refer to the relevant paper "Artemis: Articulated Neural Pets with Appearance and Motion Synthesis" for further guidance.

Artemis follows multi-view inputs. But the Data I have is monocular. Any approach now?

**Comment by chingswy** [comment]
There exists some related works: BITE: Beyond Priors for Improved ThreeD Dog Pose Estimation

**Comment by Dipankar1997161** [comment]
Yes, I saw BITE. But unfortunately the code is yet to be made available( publicly)
